src/Sylius/Component/Core/spec/Model/ProductVariantSpec.php

```diff
@@ -61,6 +71,17 @@ function its_price_should_accept_only_integer()
         $this->shouldThrow('\InvalidArgumentException')->duringSetPrice(new \stdClass());
     }
 
+    function its_original_price_should_accept_only_integer()
+    {
+        $this->setOriginalPrice(310);
+        $this->getOriginalPrice()->shouldReturn(310);
```

Comment:
I don't know if it's necessary to check this here, as we check proper values setting in ``its_original_price_should_be_mutable`` ;)